florist/api/db/entities.py

```diff
@@ -0,0 +1,33 @@
+"""Definitions for the MongoDB database entities."""
+import uuid
+from typing import Optional
+
+from pydantic import BaseModel, Field
+from typing_extensions import Annotated
```

Comment:
Not sure if I am missing something but I think annotated is available in the [standard library python typing module](https://docs.python.org/3/library/typing.html#typing.Annotated)

Reply:
Looks like the MongoDB+FastAPI tutorial I used is out of date 😅 will fix.